common/src/stack/checklist/BackendTest.py

```diff
@@ -164,23 +160,48 @@ def checkPartition(self):
 			msg = 'Backend - %s - Not Present' % BackendTest.PARTITION_XML
 			outputState("Partition_XML_Present", True, msg)
 
-	def run(self):
-		test_list = [self.checkAutoyastFiles, self.checkLudicrousStarted,  \
-				self.checkPartition, self.checkPkgInstall]
+	def checkSSHOpen(self):
+		sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+		numRetries = 100
+		count = 1
+		result = -1
+		msg = ''
+
+		while count <= numRetries:
+			result = sock.connect_ex(('127.0.0.1', 2200))
+			if result == 0:
+				outputState("SSH_Open", False, msg)
+				break
 
-		# Check for WAIT status through the install
-		chkWait = CheckWait()
-		chkWait.setDaemon(True)
-		chkWait.start()
+			count = count + 1
+			time.sleep(1)
+
+		if result == -1:
```

Comment:
Are you sure a bad result always be `-1` here?  The docs for sock.connect_ex seems to suggest it could be ERRNO, which might not always be -1?  Would it make sense to just check for != 0?